pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -770,7 +770,7 @@ class CreateVmModal extends React.Component {
             <Modal id='create-vm-dialog' show onHide={ this.props.close }>
                 <Modal.Header>
                     <Modal.CloseButton onClick={ this.props.close } />
-                    <Modal.Title> {`Create New Virtual Machine`} </Modal.Title>
+                    <Modal.Title> {`Import A Virtual Machine`} </Modal.Title>
```

Comment:
This needs the same logic as the change below - that it shows `Create New Virtual Machine` when `mode` is `create` and else it shows `Import A Virtual Machine`.

Also I just noticed that we do not translate this string. When you notice the changes in the the button text, you see that we put the strings into `_()`. Although this seems weird, `_` is a function name. This function translates strings when you use Cockpit in different language than English. Can you please also introduce these changes? Thank you.

Reply:
Yeah I'll put the similar condition on line 773 but I'm sorry I didn't understand the change regarding the button strings.

Reply:
>  I'm sorry I didn't understand the change regarding the button strings.

Notice difference between `_("Create")` and `"Create`. The second string is not translated. When you look at those texts in buttons they are translated. When you look at those titles, they are not.

Reply:
Yeah, I got that. So we don't need to translate as we're writing it only in English, right?

Reply:
not really, this is for when someone is using cockpit in different language than English. When the user uses English, then this string is presented as is - but when they pick different language, then we can translate those strings. And we (as developers) mark strings which should be translated into different languages with this construction. And we want to translate also those titles.